.github/ISSUE_TEMPLATE/flaky_test.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Flaky Test
+about: A template for Druid CI flaky test reports
+title: "Flaky Test: test name"
+labels: Flaky Test
+assignees: ''
+
+---
+
+Please replace 'test name' in the title with the actual test name. Also include a link to a failing test run, along with any investigation you might have done to triage the issue.
```

Comment:
Perhaps a stack trace too? (will be helpful if the GHA links expire after a certain time)